ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEeaSendTransactionIntegrationTest.java

```diff
@@ -341,15 +340,15 @@ void signTransactionWhenGasPriceIsNull() {
     setUpEthNodeResponse(
         request.ethNode(sendRawTransactionRequest), response.ethNode(sendRawTransactionResponse));
 
-    sendRequestThenVerifyResponse(
+    sendPostRequest(
         request.ethSigner(sendTransactionRequest), response.ethSigner(sendRawTransactionResponse));
 
     verifyEthNodeReceived(sendRawTransactionRequest);
   }
 
   @Test
   void invalidParamsResponseWhenGasPriceIsNaN() {
-    sendRequestThenVerifyResponse(
+    sendPostRequest(
```

Comment:
nit: is this "sendPostRequest" or is it "postRequestThenExpect"? (more worried about sending and posting at the same time)

Reply:
This method doesn't do any verify. So it could just be postRequest, is that clearer?